README.md

```diff
@@ -1,64 +1,84 @@
 
 ## 🌀 Decentralized Systems
-* [Ethereum](https://ethereum.org/en/developers/docs/): Decentralized world computer for smart contract development.
-* [Klaytn](https://docs.klaytn.foundation/): The metaverse blockchain for all.
+
+- [Ethereum](https://ethereum.org/en/developers/docs/): Decentralized world computer for smart contract development.
+- [Klaytn](https://docs.klaytn.foundation/): The metaverse blockchain for all.
 
 ## 👬 DAOs
-* [PangeDAO](https://www.pangeadao.org/): Metaverse land investment and development.
+
+- [PangeDAO](https://www.pangeadao.org/): Metaverse land investment and development.
 
 ## 🏦 DeFI
-* [Metapoly](https://www.metapoly.org/): A metaverse bank that creates liquidity and utility for Metaverse assets.
+
+- [Metapoly](https://www.metapoly.org/): A metaverse bank that creates liquidity and utility for Metaverse assets.
 
 ## 🕋 Marketplace
-* [Parcel](https://parcel.so/): The Marketplace for the next generation of gaming in the virtual world.
-* [SandStorm](https://app.sandstorm.co/): A marketplace for connecting brands with metaverse builders.
-* [Metahood](https://metahood.xyz/): Discover, buy & sell land across the open metaverse.
-* [Metamundo](https://www.metamundo.co/): Showcase your 3D metaverse work and connect with collectors.
+
+- [Parcel](https://parcel.so/): The Marketplace for the next generation of gaming in the virtual world.
+- [SandStorm](https://app.sandstorm.co/): A marketplace for connecting brands with metaverse builders.
+- [Metahood](https://metahood.xyz/): Discover, buy & sell land across the open metaverse.
+- [Metamundo](https://www.metamundo.co/): Showcase your 3D metaverse work and connect with collectors.
 
 ## 👤 Avatar Platform
-* [Ready Player Me](https://readyplayer.me/): Cross-game avatar platform for the Metaverse.
-* [Genies](https://genies.com/): Design and create avatar ecosystem.
+
+- [Ready Player Me](https://readyplayer.me/): Cross-game avatar platform for the Metaverse.
+- [Genies](https://genies.com/): Design and create avatar ecosystem.
 
 ## 🚀 VR,AR,MR
-* [Over the Reality](https://www.overthereality.ai/): A mixed reality metaverse that merges physical and virtual worlds.
-* [Immersed](https://immersed.com/): Work Faster in VR Than in Real Life.
+
+- [Over the Reality](https://www.overthereality.ai/): A mixed reality metaverse that merges physical and virtual worlds.
+- [Immersed](https://immersed.com/): Work Faster in VR Than in Real Life.
 
 ## 👷 Builders and Architects
-* [LandVault](https://landvault.io/): An end-to-end Metaverse Builders
+
+- [LandVault](https://landvault.io/): An end-to-end Metaverse Builders
 
 ## 🛡 Standards
-* [Metaverse Standard Forum](https://metaverse-standards.org/): A forum to develop standards for an open and inclusive metaverse.
+
+- [Metaverse Standard Forum](https://metaverse-standards.org/): A forum to develop standards for an open and inclusive metaverse.
+
+## 🎙️ Podcast
```

Comment:
Awesome work chief!

Buh can we make it metaverse-focused just like  this https://www.metaverse.fm/
And also you could google search "metaverse podcast" and filter through